environment.yml

```diff
@@ -6,15 +6,14 @@ channels:
   - rmg
   - cantera
   - pytorch
-  - ampayne
   - anaconda
   - conda-forge
 dependencies:
   - python
-  - rmg == 3.0.0b
+  - rmg == 3.0.0
   - ase
   - cclib == 1.6.2 
   - py3dmol
-  - rdkit
+  - rdkit == 2019.03.4
```

Comment:
Is this specificity necessary? Can it be >= ?

Reply:
I tried before using a newer version of rdkit but I ran into errors. However, that was a while ago which is why it's set to 2019.03.4. I'll look into newer versions real quick.

Reply:
Just tried with the latest rdkit (2019.09.02) and things are working find so I changed the env file to be >= rather than ==